app/api/v_sphere/virtual_machine.rb

```diff
@@ -58,9 +58,10 @@ def self.find_by_name(name)
     end
 
     def self.prepare_vm_names
-      return [] unless File.exist?(File.join(Puppetscript.puppet_script_path, 'Node'))
+      node_file = Puppetscript.nodes_path
```

Comment:
Should this not be node_dir/node_path instead of node_file?

Reply:
Already fixed.